tools/test/collect_cc_coverage.sh

```diff
@@ -113,7 +113,8 @@ function lcov_coverage() {
 function gcov_coverage() {
   local output_file="${1}"; shift
 
-  touch "$output_file"
+  # We'll save the standard output of each the gcov command in this log.
+  local gcov_log="$output_file.gcov.log"
 
   # Move .gcno files in $COVERAGE_DIR as the gcda files, because gcov
   # expects them to be under the same directory.
```

Comment:
Below, "If the gcda file was not found we generate empty coverage from the gcno file.", where does it happen?

Reply:
I rephrased it - if the gcda is not found then we skip generating any coverage for this gcno.